python-package/README.rst

```diff
@@ -26,7 +26,7 @@ Compiled library that is included in the wheel file supports both **GPU** and **
 
 For **Windows** users, `VC runtime <https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads>`_ is needed if **Visual Studio** (2015 or newer) is not installed.
 
-For **Linux** users, **glibc** >= 2.14 is required. Also, in some rare cases, when you hit ``OSError: libgomp.so.1: cannot open shared object file: No such file or directory`` error during importing LightGBM, you need to install OpenMP runtime library separately (use your package manager and search for ``lib[g|i]omp`` for doing this).
+For **Linux** users, **glibc** >= 2.28 is required. Also, in some rare cases, when you hit ``OSError: libgomp.so.1: cannot open shared object file: No such file or directory`` error during importing LightGBM, you need to install OpenMP runtime library separately (use your package manager and search for ``lib[g|i]omp`` for doing this).
```

Comment:
I think this is for version > 3.3.3, should we add a note about the minimal version of glibc for version <= 3.3.3 ?

Reply:
Sure!

I added this note inline in https://github.com/microsoft/LightGBM/pull/5580/commits/8def665466fd83ce7c9c7a61b586509c5c2f2db0.

The next time we change this floor, it should probably be replaced with a table similar to those in the description of #5514.